dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -10,335 +10,98 @@ nav_section: prebid-amp
 
 <div class="bs-docs-section" markdown="1">
 
-# Show Prebid Ads on AMP Pages (Alpha)
+# Show Prebid Ads on AMP Pages
 {: .no_toc}
 
-{% include amp-deprecation-warning.md %}
-
 This page has instructions for showing ads on AMP pages using Prebid.js.
 
-These instructions assume you have read [How Prebid on AMP Works]({{site.github.url}}/dev-docs/how-prebid-on-amp-works.html).
+At a high level, our implementation uses [Prebid Server][PBS] to talk to [AMP ads][AMPads] using the [AMP Real Time Config (RTC)][RTC-Overview] protocol.
+
+For more information about AMP RTC, see:
+
++ [Hpw Prebid on AMP Works]({{site.baseurl}}/dev-docs/how-prebid-on-amp-works.html)
++ [AMP RTC Overview][RTC-Overview]
++ [AMP RTC Publisher Integration Guide](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/rtc-publisher-implementation-guide.md)
 
-For more information about the techniques used here, see the sections "Running ads from a custom domain" and "Enhance incoming ad configuration" of the [AMP for Ads](https://www.ampproject.org/docs/reference/components/amp-ad) docs.
+{: .alert.alert-success :}
+For ad ops setup instructions, see [Setting up Prebid for AMP in DFP]({{site.github.url}}/adops/setting-up-prebid-for-amp-in-dfp.html).
 
 * TOC
 {:toc }
 
 ## Prerequisites
 
-To set up Prebid.js to serve ads into your AMP pages, you'll need:
+To set up Prebid to serve ads into your AMP pages, you'll need:
```

Comment:
Let's update the "Prerequisites" to the following:

To set up Prebid to serve ads into your AMP pages, you'll need:

1. An account with a Prebid Server instance
2. One or more Prebid Server demand partner configurations.  A configuration is essentially just a list of the demand partners that you want to work with, along with their respective parameters.
3. An AMP page containing at least one amp-ad element for an AMP ad network that supports Fast Fetch and AMP RTC.